python/ray/serve/deployment_state.py

```diff
@@ -52,6 +53,13 @@ class ReplicaStartupStatus(Enum):
     FAILED = 4
 
 
+class HealthCheckResponse(Enum):
```

Comment:
Can we call this Enum "ReplicaHealthCheckResponse," so we can differentiate it from the overall deployment's health status?